ui/src/App.tsx

```diff
@@ -85,15 +95,7 @@ function App() {
             },
           })}
         >
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
+          {[...Array(9).keys()].map(entry => gridButton(entry))}
```

Comment:
This isn't how you call react components. Instead, you treat them like HTML tags:

`<GridButton key={entry} />`